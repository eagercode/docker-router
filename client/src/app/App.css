.App {
    text-align: center;
}

.App-header {
    background-color: #222;
    padding: 1px;
    color: white;
}
