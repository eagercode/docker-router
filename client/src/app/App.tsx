import * as React from 'react';
import Dashboard from '../components/dashboard/Dashboard';
import './App.css';
import { AppBar } from 'material-ui';

export default class App extends React.Component<{}, {}> {
    render() {
        return (
            <div className="App">
                <AppBar
                    iconElementLeft={<span/>}
                    title="Docker Router"
                />
                <Dashboard/>
            </div>
        );
    }
}
