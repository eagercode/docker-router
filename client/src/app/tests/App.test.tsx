import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { getMuiTheme } from 'material-ui/styles';
import App from '../App';

describe('<App/>', () => {
    it('app snapshot', () => {
        const component: ShallowWrapper<App, {}> = shallow(
            <App/>,
            {
                context: {
                    muiTheme: getMuiTheme(),
                }
            }
        );

        expect(component).toMatchSnapshot();
    });
});
