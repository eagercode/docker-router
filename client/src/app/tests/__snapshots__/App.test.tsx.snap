// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> app snapshot 1`] = `
<styled.div>
  <AppBar
    iconElementLeft={<GitHub />}
    showMenuIconButton={true}
    title="Docker Router"
    zDepth={1}
  />
  <Dashboard />
</styled.div>
`;
